intro2.md

```diff
@@ -0,0 +1,405 @@
+# The Optimistic Ethereum Specification Overview
+
+Optimistic Ethereum is an _EVM equivalent_, _optimistic rollup_ protocol
+designed to _scale Ethereum_ while remaining maximally compatible with existing
+Ethereum infrastructure. This document provides an overview of the protocol to
+provide context for the rest of the specification.
+
+## Table of Contents
+
+1. [Foundations](#foundations)
+1. [Network Participants](#network-participants)
+1. [User Transactions and The Sequencer](#user-transactions-and-the-sequencer)
+1. [L2 Batches and Blocks](#l2-batches-and-blocks)
+1. [L2 Block Properties](#l2-block-properties)
+1. [L1 Components Overview](#l1-components-overview)
+   - [Data Feeds](#data-feeds)
+   - [Fraud Proof Manager](#fraud-proof-manager)
+
+## Foundations
+
+### What is Ethereum scalability?
+
+Ethereum's limited resources, specifically bandwidth, computation, and storage,
+constrain the number of transactions which can be processed on the network,
+leading to extremely high fees. Scaling Ethereum means increasing the number of
+useful transactions the Ethereum network can process, by increasing the supply
```

Comment:
Scalability is increasing throughput without increasing the hardware requirements, see https://twitter.com/_prestwich/status/1284174486674083840?s=20

Reply:
Rollups by definition increase the throughput of the system. Users that care about validating the state can opt into validating the state

Reply:
wontfix lol